docs/getting-started/getting-started-with-notebooks.rst

```diff
@@ -8,131 +8,122 @@ Notebooks
 Tutorial Goals
 --------------
 
-- Get familiarized with the concept of notebooks on Dockstore
-- Learn the basics of creating a notebook using a .dockstore.yml file
-- Learn how to register a notebook
-- Publish your notebook to Dockstore
+- Create a notebook
+- Save a notebook to GitHub
+- Register a notebook on Dockstore
+- Publish a notebook on Dockstore
 
-Notebooks are documents that can include code, text, images, and other media resources. They can be executed in notebook environments like Google Colab and JupyterLab.
-This document outlines how to register, update, and publish a notebook on Dockstore.
+Notebooks are documents that include code, text, images, and other media resources, typically in subsections called "cells".  You can create, modify, and execute notebooks in browser-based, interactive environments like Google Colab and JupyterLab.  This document outlines how to create, save, register, update, and publish a notebook on Dockstore.
 
 Overview
 --------
 
-In this tutorial, you will register a notebook using the Dockstore GitHub App, which automatically syncs your notebook from GitHub to Dockstore. More information about the Dockstore GitHub App can be found :doc:`here </getting-started/github-apps/github-apps-landing-page>`.
+In this tutorial, you will create a notebook, save it to GitHub, and register your notebook on Dockstore using our GitHub App, which updates it when your repository changes.  More information about the Dockstore GitHub App can be found :doc:`here </getting-started/github-apps/github-apps-landing-page>`.
 
-Dockstore Notebooks takes advantage of Google Colab's integration with GitHub. The integration allows you to develop your notebook in Google Colab and push your changes to GitHub from Google Colab. With the Dockstore GitHub App installed on your repository, any edits saved to your notebook from Google Colab will automatically be synced to Dockstore.
+To create your notebook, you'll use `Google Colab <https://colab.research.google.com/>`_, a popular online notebook service.  Of course, you can author notebooks in other environments, such as `GitHub Codespaces <https://github.com/features/codespaces>`_, `MyBinder <https://mybinder.org/>`_, or `Jupyter <https://jupyter.org/try-jupyter/lab/>`_, but Colab is a clean, easy-to-use option that's perfect for beginners.
 
-Setting up the Google Colab and GitHub integration
---------------------------------------------------
+.. note:: If you've already created a notebook, but it's not on GitHub, start at :ref:`Save Your Notebook to GitHub <save-your-notebook-to-github>`.  If your notebook is on GitHub, skip to :ref:`Create Your .dockstore.yml File <create-your-dockstore-yml-file>`.
```

Comment:
The Save Your Notebook to GitHub section is Google Colab focused even though users may have authored it in another environment like you mentioned above. Maybe we could have a sentence or two about how to save a notebook to GitHub for other authoring environments in that section? (download notebook, commit and push to your repo?)